anxcloud/schema_object_storage.go

```diff
@@ -0,0 +1,336 @@
+package anxcloud
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	objectstoragev2 "go.anx.io/go-anxcloud/pkg/apis/objectstorage/v2"
+)
+
+// schemaObjectStorageCommon returns the common schema fields for all Object Storage resources
+func schemaObjectStorageCommon() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"customer_identifier": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "Identifier of the customer the resource should be assigned to (mandatory).",
+		},
+		"reseller_identifier": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Identifier of the reseller the resource should be assigned to.",
+		},
```

Comment:
I think we never exposed those fields as they're not visible from a customer perspective if I'm not mistaken.